CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 * [ENHANCEMENT] Ingester/Querier: Optimise regexps with long lists of alternates. #8221, #8234
 * [ENHANCEMENT] Ingester: Include more detail in tracing of queries. #8242
 * [EHNAHCEMENT] Distributor: add `insight=true` to remote-write and OTLP write handlers when the HTTP response status code is 4xx. #8294
+* [BUGFIX] Map Non-Retryable Errors to Retryable in OTLP Endpoint. #8324
```

Comment:
This makes little sense to me. Can we explain that OTLP endpoint currently uses codes that would be retry-able in Prometheus remote-write protocol, but are not retryable in OTLP. This PR fixes the mapping.

Reply:
[Nit] I also don't see the point of title case in a changelog item.